### DISCUSSION TITLE
Degrees of Freedom

### DISCUSSION THREAD

**Discussion by safdarmt** [cooperrc/advanced-dynamics#29] [General]
<img width="719" alt="Screenshot 2024-02-07 at 2 35 49 PM" src="https://github.com/cooperrc/advanced-dynamics/assets/138720786/898e1b5d-40be-4141-8619-c5fe4e16d40c">

Here in the degree of freedom example from a lecture, it says that the cube dropped into a box will still have 6 DOF. I am a little confused because isn't the bottom of the box a planar surface? Therefore, should the cube not have 3 DOF instead?

**Comment by SentientBiped** [comment]
This example is a die being thrown into the box and bouncing around so after it bounces you would still have 6 DOF to determine the motion of the die.

**Comment by C-Olsen** [comment]
Muhammad, a thrown dice is a rigid body in motion.  All rigid bodies have 6 DOFs.  Easiest to picture in a cartesian coordinate system.  Ability to move in 3 dimensions (X, Y, Z) and the ability to rotate about each of those axis, (RX, RY, RZ).

An airplane is another easy way to picture the concept.  It can move forward/back, side to side, and up and down.  It can also rotate three ways pitch, yaw and roll.  6 DOFs total.
